cookbooks/prescribed_velocity/prescribed_velocity.cc

```diff
@@ -120,7 +165,11 @@ namespace aspect
   {
     if (prescribe_internal_velocities)
       {
-        Quadrature<dim> quadrature (simulator_access.get_fe().get_unit_support_points());
+        const Parameters<dim> parameters = simulator_access.get_parameters();
+        const bool use_locally_conservative_discretization = parameters.use_locally_conservative_discretization;
+
+        std::vector< Point<dim> > points = get_unit_support_points_for_velocity<dim>(simulator_access,use_locally_conservative_discretization);
```

Comment:
* no need to explicitly say `<dim>` in the function call.

* mark the variable as `const`

* also, can you break at the comma between function arguments? The line is long enough.